inc/c_util/constbuffer_array_splitter_array.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef CONSTBUFFER_ARRAY_SPLITTER_ARRAY_H
```

Comment:
>CONSTBUFFER_ARRAY_SPLITTER_ARRAY_H [](http://example.com/codeflow?start=8&length=34)

maybe this file should be called `constbuffer_array_tarray` instead

Reply:
done